src/main/typescript/node_modules/@atomist/rug/operations/RugOperation.ts

```diff
@@ -102,9 +102,28 @@ class ReviewResult {
     this.comments.push(rc)
   }
 
-  static empty(note: string) {
+  static empty(note: string): ReviewResult {
```

Comment:
With typical import usage (`import {ReviewResult} from "RugOperation"`) it will read `ReviewResult.empty()` so that would seem duplicative.